packages/react-native-bpk-component-badge/src/BpkBadgeIcons.js

```diff
@@ -47,10 +50,10 @@ const BpkBadgeIcons = (props: Props) => {
     itemStyle.push(StyleSheet.flatten(userItemStyle));
   }
 
-  const adjustedIcons = icons.map(icon =>
+  const adjustedIcons = icons.map((icon, index) =>
     React.cloneElement(icon, {
       small: true,
-      style: itemStyle,
+      style: [itemStyle, index === icons.length - 1 ? styles.lastItem : null],
```

Comment:
Where are your css pseudo selectors when you need em? :P

Reply:
I know right!